src/explore-education-statistics-admin/src/pages/release/data/__tests__/ReleaseApiDataSetDetailsPage.test.tsx

```diff
@@ -371,6 +372,30 @@ describe('ReleaseApiDataSetDetailsPage', () => {
     ).not.toBeInTheDocument();
   });
 
+  test('does not render the create new version button when release series includes data set version', async () => {
```

Comment:
Is this the right wording? Isn't it checking for the data set previous release ids, not the data set version? (Same question for the live table version of this test below).

Reply:
Hi there, I've reworded slightly to:

`does not render the create new version button when release series includes a previous version of this data set`

and

`does not render 'Create new version' buttons when release series contains previous version of this data set`

Is this clearer?  Thanks!